ContactDetailsApi.Tests/ContactDetailsApi.Tests.csproj

```diff
@@ -27,13 +27,13 @@
         <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
     <PackageReference Include="FluentAssertions" Version="5.10.3" />
-    <PackageReference Include="Hackney.Core.DynamoDb" Version="1.81.0" />
+    <PackageReference Include="Hackney.Core.DynamoDb" Version="1.82.0-feat-bump-dynamo0002" />
     <PackageReference Include="Hackney.Core.Sns" Version="1.52.0" />
     <PackageReference Include="Hackney.Core.Testing.DynamoDb" Version="1.57.0" />
     <PackageReference Include="Hackney.Core.Testing.Shared" Version="1.54.0" />
     <PackageReference Include="Hackney.Core.Testing.Sns" Version="1.71.0" />
     <PackageReference Include="Hackney.Shared.Person" Version="0.14.0" />
-    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="5.0.3">
+    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="6.0.0">
```

Comment:
From their documentation:
`You do not need to manually install this NuGet package to your project if you are using .NET5 SDK or later. `

But since we have it specified explicitly, we may as well use the latest stable version, which is `8.0.0`.